e33lin/grocery-store
# all recs page

- connected rails to python script (dummy script for now, will replace with the actual search script in another task)
- parses for the data in controller
- the view shows off the links to the stores

UPDATE: added a reference to PWD environment variable (which i am hoping is automatically generated for everyone) so the full path should be correctly called now for each person. test it and let me know:

### testing

- `rails server`
- go to /recommendations
- make sure there are no errors and the links to store 1, store 2, and store 3 are clickable

![Screen Shot 2023-01-24 at 11 46 42 PM](https://user-images.githubusercontent.com/65259395/214482072-6a81c2f0-2990-407a-8b1e-9afbf739636b.png)
